src/data-dictionary/events/NrMTDConsumption/NrMTDConsumption.event-definition.md

```diff
@@ -5,4 +5,4 @@ dataSources:
   - Account Usage
 ---
 
-This event tracks your New Relic platform usage and estimated cost across an entire billing period. It generates aggregate values from the NrConsumption event.
\ No newline at end of file
+This event tracks your New Relic usage (data and billable users) and estimated cost across an entire calendar month. It generates aggregate values from the NrConsumption event.
```

Comment:
```suggestion
This event tracks your New Relic usage (data and billable users) and estimated cost across an entire calendar month. It generates aggregate values from the `NrConsumption` event.
```

Reply:
Honestly not sure if that has to be in code or not.

Reply:
you're right that it generally should but I'm actually not sure if the data dictionary event descriptions can show code format but I'll try it.